dockstore-webservice/src/main/java/io/dockstore/webservice/resources/WorkflowResource.java

```diff
@@ -2120,6 +2121,18 @@ public Response handleGitHubInstallation(@ApiParam(hidden = true) @Parameter(hid
             triggerUser.ifPresent(lambdaEvent::setUser);
             lambdaEventDAO.create(lambdaEvent);
         });
+        // make some educated guesses whether we should try to retrospectively release some old versions
+        // note that for large organizations, this loop could be quite large if many repositories are added at the same time
+        for (String repository: repositories.split(",")) {
+            final Set<String> strings = identifyGitReferencesToRelease(repository, installationId);
+            for (String gitReference: strings) {
+                if (LOG.isInfoEnabled()) {
+                    LOG.info(String.format("Retrospectively processing branch/tag %s in %s(%s)", Utilities.cleanForLogging(gitReference), Utilities.cleanForLogging(repository),
```

Comment:
fwiw, log4j supports lambdas as arguments, so we can avoid the `isInfoEnabled`/etc checks: https://www.baeldung.com/log4j-2-lazy-logging

Reply:
https://ucsc-cgl.atlassian.net/browse/SEAB-5229